packages/big-design/src/components/Pagination/Pagination.tsx

```diff
@@ -29,17 +33,12 @@ export interface PaginationProps extends MarginProps {
   getRangeLabel?(start: number, end: number, totalItems: number): string;
 }
 
-const defaultGetRangeLabel = (
-  start: number,
-  end: number,
-  totalItems: number,
-  of = 'of',
-): string => {
+const defaultGetRangeLabel = (start: number, end: number, totalItems: number): string => {
```

Comment:
Can we add this as part of the `localization` object?

Reply:
Like having the function be part of the object? My concern is that this `getRangeLabel` can be used for non localization purposes.